src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DefaultImportDocumentSnapshot.cs

```diff
@@ -26,25 +25,19 @@ public DefaultImportDocumentSnapshot(ProjectSnapshot project, RazorProjectItem i
         _version = VersionStamp.Default;
     }
 
-    public override string FileKind => null;
-
-    public override string FilePath => null;
-
-    public override string TargetPath => null;
+    public override string FileKind => null!;
+    public override string FilePath => null!;
+    public override string TargetPath => null!;
```

Comment:
Well this sucks! Would a fake path be better? And some Debug.Asserts or something?

Reply:
I agree that it sucks! This is still a draft PR, and I intend to look at this before marking it for review.

Reply:
Sorry if I peeked early, I go off GitHub notifications in email, and didn't even notice the status.

Though it's odd GitHub emailed 🤷‍♂️

Reply:
I don't mind the early look and feedback. I just wanted to let you know that I don't want to leave these properties like this.